action.yml

```diff
@@ -0,0 +1,12 @@
+name: "Conform Action"
+description: "Policy enforcement for your pipelines."
+branding:
+  icon: "check-circle"
+  color: "gray-dark"
+inputs:
+  token:
+    description: "The GitHub token. Must have repo:status access. Can be specified by using {{ secrets.GITHUB_TOKEN }}"
```

Comment:
Should github token be used here or input token?

Reply:
token